src/services/balancer.js

```diff
@@ -1,251 +1,186 @@
 import { logger } from '../services/logger';
+import Fees from '../services/fees';
 const debug = require('debug')('router');
 const sor = require('@balancer-labs/sor');
 const BigNumber = require('bignumber.js');
 const ethers = require('ethers');
-const proxyArtifact = require('../static/ExchangeProxy.json');
+const proxyArtifact = require('../static/balancer_vault_abi.json');
 const globalConfig =
   require('../services/configuration_manager').configManagerInstance;
 
 // constants
-const MULTI = '0xeefba1e63905ef1d7acba5a8513c70307c1ce441';
-const MULTI_KOVAN = ' 0x2cc8688C5f75E365aaEEb4ea8D6a480405A48D2A';
 const MAX_UINT = ethers.constants.MaxUint256;
-const GAS_BASE = globalConfig.getConfig('BALANCER_GAS_BASE') || 200688;
```

Comment:
I think we should just let all these config go into the global config file because the `spender` function for approval and allowances currently reads from that file.